arwen/common.go

```diff
@@ -50,6 +50,7 @@ type VMHostParameters struct {
 	GasSchedule              config.GasScheduleMap
 	ProtocolBuiltinFunctions vmcommon.FunctionNames
 	ElrondProtectedKeyPrefix []byte
+	ArwenV2EnableEpoch       uint32
```

Comment:
Not sure if `arwenV2` is the version / tag we want on the long term. Perhaps `v0_4_0` or something like that, to match the release tags? It seems fine to use `v2` as well, but then we should bump the major version for Github tags, too.

Also, we should make a list of the breaking changes under this v2.

Reply:
will do on release. and integration on Elrond. it is V2 - since V1 was launched at the mainnet